### ISSUE TITLE
Add an API call to immediately halt all operations

### ISSUE THREAD

**Issue by bartv** [inmanta/inmanta#2228]
Currently when something is going wrong, the only way to quickly halt operations is click "clean". However this also removes all the logs in the database.

This ticket should add an API call to halt/resume all orchestrator functions. I think this should be added to the environment service.

Concrete:
- No compiles
- No facts refresh
- No deploys / repairs
- Abort all inflight deploys

**Comment by bartv** [comment]
@arnaudsjs iirc an agent pause finishes the current inprogress resources?

**Comment by arnaudsjs** [comment]
> @arnaudsjs iirc an agent pause finishes the current inprogress resources?

Yes. The resources which are being deployed, continue to deploy and the scheduled resources are cancelled.
